python/pyspark/sql/plot/plotly.py

```diff
@@ -27,4 +27,18 @@
 def plot_pyspark(data: "DataFrame", kind: str, **kwargs: Any) -> "Figure":
     import plotly
 
+    if kind == "pie":
+        return plot_pie(data, **kwargs)
+
     return plotly.plot(PySparkPlotAccessor.plot_data_map[kind](data), kind, **kwargs)
+
+
+def plot_pie(data: "DataFrame", **kwargs: Any) -> "Figure":
```

Comment:
I am wondering why need this method?
to map the argument name? `y` -> `values` ?

Reply:
I'm thinking that support for subplots may be added in the future. Currently only Matplotlib supports subplots, even for Pandas on Spark.

Reply:
Let me know if we prefer to remove it for now.

Reply:
let's keep it, we can support subplot later